TypeCobol.LanguageServer/TypeCobolLanguageServer.cs

```diff
@@ -278,6 +282,48 @@ public void WholeDocumentChanged(object sender, EventArgs eventArgs)
             UpdateTokensLines(compilationDocument);
         }
 
+        /// <summary>
+        /// LSR Cfg Analyzer ID.
+        /// </summary>
+        internal const string lspcfgId = "lsp-cfg";
+
+        /// <summary>
+        /// Method to update CFG/DFA information.
+        /// </summary>
+        /// <param name="fileCompiler">The underlying File Compiler</param>
+        /// <param name="lsrMode)">True if this action is executed in a LSR Mode</param>
+        /// <returns>CFG/DFA Data information</returns>
+        public CfgDfaParams UpdateCfgDfaInformation(DocumentContext docContext, bool lsrMode)
```

Comment:
About the name `lsrMode` for the boolean parameter, I think it is more something like `writeToFile` because the purpose of this method is to turn CFG/DFA results into a `CfgDfaParams` object. The method is not concerned by the mode (LSR or regular) but rather accepts an option on how to create its result. The parameter should be named from method perspective not caller perspective.

Reply:
+1

Reply:
Done in d7cc3ff